packages/frontend/editor-ui/src/stores/evaluation.store.ee.ts

```diff
@@ -22,24 +20,14 @@ export const useEvaluationStore = defineStore(
 		const pollingTimeouts = ref<Record<string, NodeJS.Timeout>>({});
 
 		// Store instances
-		const posthogStore = usePostHog();
 		const rootStore = useRootStore();
 		const workflowsStore = useWorkflowsStore();
 		const nodeTypesStore = useNodeTypesStore();
 		const settingsStore = useSettingsStore();
 
 		// Computed
 
-		// Enable with `window.featureFlags.override('032_evaluation_mvp', true)`
-		const isFeatureEnabled = computed(() =>
-			posthogStore.isFeatureEnabled(WORKFLOW_EVALUATION_EXPERIMENT),
-		);
-
-		const isEvaluationEnabled = computed(
-			() =>
-				posthogStore.isFeatureEnabled(WORKFLOW_EVALUATION_EXPERIMENT) &&
-				settingsStore.settings.evaluation.quota !== 0,
-		);
+		const isEvaluationEnabled = computed(() => settingsStore.settings.evaluation?.quota !== 0);
```

Comment:
NItpicking but should we check for > 0 here? 🤔

Reply:
As far as I remember `quota: -1` means unlimited for evaluations :)